### ISSUE TITLE
Reflection Probe Too Glossy wIth Panorama Sky

### ISSUE THREAD

**Issue by WickedInsignia** [godotengine/godot#84738]
### Godot version

4.5 Beta 5

### System information

Windows 11, Nvidia RTX4070ti, AMD Ryzen7700x

### Issue description

Reflection Probes will make all materials under their influence too glossy when a Panorama sky (.EXR or .HDR) is used.
This does not seem to affect specular reflections from light sources and reflections from other objects also appear to be the correct roughness. Only reflections from the sky seem to be too glossy.
Brightness of surrounding objects also seems to be affected (the ground seems to be too bright in the metallic material, even though the probe was updated to account for the change in lighting).

**With Panorama Sky:**
![ReflectionProbeGlossy_Panorama](https://github.com/godotengine/godot/assets/70243134/b2020bbb-8571-4855-b1ec-544dbff226c5)

**With Procedural Sky:**
![ReflectionProbeGlossy_Procedural](https://github.com/godotengine/godot/assets/70243134/99c4a465-9480-43b5-b0e9-d0205c3d9ffc)

### Steps to reproduce

- Load MPR
- Panorama Sky should be there by default and the materials are too glossy
- Change sky to Procedural or Physical
- Open a new tab and return to the main scene so the probe is updated
- Reflections should be adequate roughness

### Minimal reproduction project

[ReflectionProbeGlossiness.zip](https://github.com/godotengine/godot/files/13324646/ReflectionProbeGlossiness.zip)

**Comment by Calinou** [comment]
If you use HDRIs sourced from real life, they likely have extremely bright spots. Enable **HDR Clamp Exposure** in the import dock to clamp their exposure to something more reasonable that won't generate fireflies in the radiance map.

**Comment by WickedInsignia** [comment]
Yeah that seems to be the problem. Clamping that sky didn't reduce the effect enough to be usable, but using a HDR without any super bright areas such as the one attached below resolved the issue:
[HosekWilkieSky_Render01.zip](https://github.com/godotengine/godot/files/13329853/HosekWilkieSky_Render01.zip)

**Comment by huwpascoe** [comment]
The weird thing is that this happens only in update_mode = ONCE
![Image](https://github.com/user-attachments/assets/9b05a58a-93cd-4861-a896-1a67f70dfb03)

set it to ALWAYS and the weird artifacts are gone.

**Comment by Calinou** [comment]
> set it to ALWAYS and the weird artifacts are gone.

The **Always** update mode uses the real-time radiance filter, instead of the incremental one. It's generally lower quality, but it doesn't suffer from firefly artifacts with very bright images.

https://github.com/godotengine/godot/pull/97773 addresses this by letting you change the filter mode for each ReflectionProbe without needing to change the update mode at the same time.
